src/test/java/seedu/recipe/logic/commands/DeleteCommandTest.java

```diff
@@ -49,10 +54,10 @@ public void execute_invalidIndexUnfilteredList_throwsCommandException() {
 
     @Test
     public void execute_validIndexFilteredList_success() {
```

Comment:
could maybe add comments to detail what this method is doing (filter recipe list first, then delete a recipe, and assert that the result is an empty recipe book)

Reply:
fixed